workspaces/liferay-sample-workspace/client-extensions/liferay-sample-editor-config-contributor/client-extension.yaml

```diff
@@ -3,6 +3,7 @@ assemble:
       into: static
 liferay-sample-editor-config-contributor:
     editorConfigKeys:
+        - contentEditor
```

Comment:
I don't know any Alloy Editor usage under FI responsibility, so I chose Blogs, my typical go-to for checking AE. Applying sample to Blogs or some similar location is still probably better than building a new AE sample module.

Reply:
I'm fine tiwh this. I don't think we have to invest in a AE sample in 2024